src/main/java/mods/eln/node/AutoAcceptInventoryProxy.kt

```diff
@@ -19,30 +18,26 @@ class AutoAcceptInventoryProxy(val inventory: IInventory) {
     }
 
     private abstract class ItemAcceptor(val index: Int) {
-        abstract fun take(itemStack: ItemStack, inventory: IInventory): Boolean
+        abstract fun take(itemStack: ItemStack?, inventory: IInventory): Boolean
     }
 
     private open class ItemAcceptorIfEmpty(index: Int, val acceptedItems: Array<out Class<out Any>>)
         : ItemAcceptor(index) {
-        override fun take(itemStack: ItemStack, inventory: IInventory): Boolean {
-            // Do nothing if we already have a stack.
-            if (inventory.getStackInSlot(index).isEmpty) {
+        override fun take(itemStack: ItemStack?, inventory: IInventory): Boolean {
+            if (inventory.getStackInSlot(index) == null) {
```

Comment:
Don't remove comments. I question the logical change here, you may need to be checking both.